src/app/containers/ImageWithPlaceholder/index.jsx

```diff
@@ -13,7 +14,7 @@ const renderImage = (imageToRender, lazyLoad, fallback) =>
       <LazyLoad offset={LAZYLOAD_OFFSET} once>
         {imageToRender}
       </LazyLoad>
-      {fallback && <noscript>{imageToRender}</noscript>}
+      {fallback && <noscript>{renderToString(imageToRender)}</noscript>}
```

Comment:
I'm ok with this if @dr3 is ok with it!

Reply:
Discussed IRL, as bundle size increases an alternative solution has been implemented.